### PR TITLE
Rename tests to default convention (Part 1)

### PR DESCRIPTION
**Pull request by landim**
Refactoring the integration tests to same convention:

> Should_State_When_Action

### CODE REVIEW

File: src/Cassandra.IntegrationTests/Core/ClusterTests.cs [hunk 1]
```diff
@@ -24,6 +24,26 @@ public void TestTearDown()
...
+        /// @since 3.0.0
+        /// @jira_ticket CSHARP-378
+        /// @expected_result The default consistency level should be LOCAL_ONE
+        /// 
+        /// @test_category consistency
+        [Test]
+        public void LocalOne_Is_Default_Consistency()
+        {
+            //TODO refactor/move this test
+//            _testCluster = TestClusterManager.CreateNew(2);
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Core/ClusterTests.cs]: Commented line :)

**Comment by landim** [comment]
This is the first part of the integration tests renaming pattern and improvements.

**Comment by landim** [comment]
The test failed is regarding an issue with ccm multinode.

File: src/Cassandra.IntegrationTests/Core/ClientWarningsTests.cs [hunk 2]
```diff
@@ -36,6 +37,30 @@ public void SetupFixture()
             Session.Execute(String.Format(TestUtils.CreateTableSimpleFormat, Table));
         }
 
+        [Test]
+        public void QueryTraceEnabledTest()
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Core/ClientWarningsTests.cs]: Nitpicking: name of the moved test.

File: src/Cassandra.IntegrationTests/Core/ClientWarningsTests.cs [hunk 3]
```diff
@@ -36,6 +37,30 @@ public void SetupFixture()
...
+                Assert.NotNull(rs.Info.QueryTrace.ClientAddress);   
+            }
+            else
+            {
+                Assert.Null(rs.Info.QueryTrace.ClientAddress);
+            }
+        }
+
+        [Test]
+        public void QueryTraceDisabledByDefaultTest()
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Core/ClientWarningsTests.cs]: Same.

**Pull request merged**